packages/maskbook/src/plugins/Valuables/UI/BidCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { useAsync } from 'react-use'
+import { makeStyles, Typography, Button } from '@material-ui/core'
+import { getTweetBid } from '../api'
+
+// import { ETHIcon } from '@dimensiondev/icons'
+// <i class="fab fa-ethereum"></i>
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'flex-end',
+    },
+    bidButton: {
+        borderRadius: '20px',
+        backgroundColor: theme.palette.mode === 'light' ? '#EFF1F2' : '#1E2024',
+        width: '100vw',
```

Comment:
hmm are you sure a button for 100 view port width?

Reply:
Changed it to 100% width, do you mean you're unsure of using the button component there or that it's too wide?